CoreVisualSchema/src/au/gov/asd/tac/constellation/graph/schema/visual/concept/VisualConcept.java

```diff
@@ -269,6 +269,11 @@ public String getName() {
                 .setDefaultValue(0.0f)
                 .create()
                 .build();
+        public static final SchemaAttribute POSITIONLOCKED = new SchemaAttribute.Builder(GraphElementType.VERTEX, BooleanAttributeDescription.ATTRIBUTE_NAME, "position_locked")
```

Comment:
A minor one again but can this be called `POSITION_LOCKED` instead?

Reply:
Ive actrually refactored to use the terms PINNED and UNPINNED to better represent icons used.